ondemand.osc.edu/apps/myjobs/templates/Basic_AMBER_Job_Owens/basic_amber.sh

```diff
@@ -1,22 +1,61 @@
-#PBS -N 6pti
-#PBS -l nodes=1:ppn=28
-#PBS -l walltime=0:20:00
+# AMBER 16 Example Batch Script for Owens
+#
+#PBS -N jac9999cuda.owens
+#PBS -j oe
+#PBS -m ae
+#PBS -M srb@osc.edu
```

Comment:
I think that we don't want to have Scott get emails every time people submit jobs :-)